cms/__init__.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
Does this commit makes the previous one ($(which python) in travis) useless?

Reply:
The shebang only affects executions of the form `./foo.py`, whereas we used to explicitly invoke `python prerequisites.py`. I tried changing the Travis config to use `./prerequisites.py` and it does indeed work, however I am not sure it is doing what we expect: it is invoking a Python 3 interpreter, but probably not the one we think.

On Travis, `sudo` doesn't fully preserve the environment. Since we are in a venv, the `PATH` is modified so that `python` refers to the venv's Python interpreter. As proven by the bug I'm fixing here, within `sudo`, `python` resolves to Python 2: this means that inside `sudo` a different `PATH` is being used. (I haven't been unable to replicate this issue locally, I guess Travis is using a custom config for sudo and/or venvs). We've already faced this issue in f1d5955e8365827a15c57af89f06c5c5b1824de4.

If we now used `./prerequisites.py` with the `python3` shebang my guess is that we would be using the system Python 3 interpreter, not the venv one. I don't know what version the system interpreter is, but since it's Ubuntu 14.04 (trusty), it may very well be 3.4 which, although `prerequisites.py` doesn't do anything fancy, we probably don't want to use.

I may be totally wrong, but I think it is safer to stick with `$(which python)`.